src/transaction.local.net.spec.ts

```diff
@@ -15,8 +16,8 @@ describe("test transaction", function () {
         ({ alice, bob } = await loadTestWallets());
     });
 
-    it("should send transactions", async function () {
-        this.timeout(30000);
+    it("should send transactions and wait for completion", async function () {
+        this.timeout(70000);
```

Comment:
Yup. Because the completion checks are now much more strict, https://github.com/multiversx/mx-chain-proxy-go/pull/386, and the wait time is increased.

Let's add links towards the following PRs in the description:
 - https://github.com/multiversx/mx-chain-proxy-go/pull/386
 - https://github.com/multiversx/mx-sdk-js-network-providers/pull/41